src/executePrettier.js

```diff
@@ -37,7 +47,15 @@ const executePrettier = (editor, text) => {
           filePath: getCurrentFilePath(editor),
         }));
     }
-    return prettier.format(text, getPrettierOptions(editor));
+
+    const prettierOptions = getPrettierOptions(editor);
+    const localPrettier = getLocalPrettierPath(getCurrentFilePath(editor));
+
+    if (!localPrettier) {
+      return prettier.format(text, prettierOptions);
+    }
+
+    return getLocalPrettier(localPrettier).format(text, prettierOptions);
```

Comment:
Code style comment: I think there's more than one responsibility going on in this function. This function is supposed to be responsible for executing prettier, but now it also has the responsibility for figuring out which prettier to use and where to get it.

I think it would be better, therefore, if we made a helper method simply called `getPrettier` that would handle all of this internally. This would also make mocking much easier because we can just mock `getPrettier`.